src/Exception/NotExposedHttpException.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Exception;
+
+use Symfony\Component\HttpKernel\Exception\HttpException;
+
+/**
+ * @author Vincent Chalamon <vincentchalamon@gmail.com>
+ */
+class NotExposedHttpException extends HttpException
```

Comment:
Why not extending the `NotFoundHttpException`?

Reply:
WDYT @dunglas ?